rules/warn-then-call/rule.md

```diff
@@ -0,0 +1,81 @@
+---
+type: rule
+title: Calling - Do you check your contact is free before calling them?
+uri: warn-then-try-calling-them
+authors:
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+  - title: The Sizzler (Piers Sinclair)
+  url: https://ssw.com.au/people/piers-sinclair
+  - title: Ulysses Maclaren
+  url: https://www.ssw.com.au/people/uly
+  - title: Jonty Gardner
+  url: https://www.ssw.com.au/people/jonty-gardner
+related:
+  - no-hello
+  - appointments-throw-it-in-their-calendar
+redirects:
+  - im-your-contact-to-see-if-they’re-free-before-you-try-calling-them
+  - calling-do-you-im-your-contact-to-see-if-theyre-free-before-you-try-calling-them
+  - calling-do-you-im-your-contact-to-see-if-they’re-free-before-you-try-calling-them
+  - im-your-contact-to-see-if-theyre-free-before-you-try-calling-them
+  - warn-then-try-calling-them
+created: 2015-10-05T05:50:19.000Z
+archivedreason: null
+guid: f1f96132-3b5d-43a6-9ed4-aed16f885875
+---
+It can be very jarring when somebody is called out of the blue and they are not expecting it. They might be deep in a task or talking to a client and by calling them their focus is getting disrupted.
+
+A good way to initiate a call is to warm them up by giving a warning. E.g. *“Calling you in 1 min to talk about the Northwind production site being down 💀.”*
+
+If the conversation has history, like a PBI or email, then it's important to include that context.
+
+`youtube: https://www.youtube.com/embed/oOEpdViHeVI`
+**Video: Boost Productivity: Say Goodbye to Time Wasting with this Simple Trick - No Hello | SSW Rules (6 min)**
+
+<!--endintro-->
+
+## Benefits of a Warm Call:
+
+* **Prioritization** - The other person can see what the call is about and can determine if it is a priority. For example, if you mention that you want to talk about an email from several years ago, they might think it isn't that important. Conversely, if you mention the production server is down, they will almost definitely answer you!
+* **Consideration** - It gives the other person the time to prepare. They might be busy or in another call.
+* **Easier Responses** - It puts you into their chat history, so even if they decline the call, they can easily write back to you to explain why.
+* **Availability** - Giving someone even a 1 minute warning is a good idea even if you see their Teams Status is on green ‘available’. Remember it is not easy to always change their status to ‘busy’ - they could be on a mobile phone, on a WhatsApp call, in a physical meeting, etc.
+
+:::note
+Remember that if someone's Teams status suggests that they're busy (Do not Disturb, Presenting, or On a call) make sure not to call them unless it's an emergency.
+:::
```

Comment:
Hey @SamBWagner we don't have a `note` box
See the avaliable boxes here https://ssw.com.au/rules/rule/#5-boxes

@tiagov8 - Make an issue if you think `note` would be useful 

```suggestion
**⚠️ Warning**: 
First check the persons Teams status is not "Do not Disturb", "Presenting", or "On a call" make sure not to call them unless it's an emergency.
```